internal/pkg/cli/svc_init.go

```diff
@@ -253,6 +262,9 @@ func (o *initSvcOpts) Ask() error {
 	if err := o.validateSvc(); err != nil {
 		return err
 	}
+	if err := o.askIngressType(); err != nil {
```

Comment:
I feel like the #dream setup would be separate commands `initLBWSvcOpts`, `initBackendSvcOpts`, `initWorkerSvcOpts`, `initRDWSvcOpts` that all implement the same `Validate(), Ask(), Execute()` interface and this command just does the switchin based on the type

Reply:
2. Do we not need to update `Validate` to ensure the customer's input is one of the valid values?